src/main/java/com/google/devtools/build/lib/rules/platform/PlatformRule.java

```diff
@@ -130,6 +131,13 @@ public RuleClass build(RuleClass.Builder builder, RuleDefinitionEnvironment env)
             attr(REQUIRED_SETTINGS_ATTR, BuildType.LABEL_LIST)
                 .allowedRuleClasses("config_setting")
                 .allowedFileTypes(FileTypeSet.NO_FILE))
+        /* <!-- #BLAZE_RULE(platform).ATTRIBUTE(no_toolchain_error) -->
+        A custom error message that is displayed when a required toolchain type cannot be found for this platform. Intended to point to relevant documentation users can read to understand why their toolchains are misconfigured.
```

Comment:
```suggestion
        A custom error message that is displayed when a mandatory toolchain requirement cannot be satisfied for this target platform. Intended to point to relevant documentation users can read to understand why their toolchains are misconfigured.
```

Nit: one might otherwise wonder whether this is about invalid references to `toolchain_type` targets. Clarifying that this is about toolchains for a target platform makes it more obvious which role the platform plays in this.